expression/builtin_encryption_vec.go

```diff
@@ -248,11 +250,85 @@ func (b *builtinSHA2Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column)
 }
 
 func (b *builtinCompressSig) vectorized() bool {
-	return false
+	return true
+}
+
+var (
+	bytePool = sync.Pool{
+		New: func() interface{} {
+			return make([]byte, defaultChunkSize)
+		},
+	}
+)
+
+func allocByteSlice(n int) []byte {
+	if n > defaultChunkSize {
```

Comment:
It not reasonable to use `defaultChunkSize` as a threshold.

Reply:
i use local parameter

Reply:
Yes I know, but `defaultChunkSize` is used for the chunk.column related pool, not for this.